inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -212,6 +212,28 @@ def _is_core(record):
         obj.extra_data['core'] = True
 
 
+def get_wfs_not_completed(obj, skip_blocked=True):
+    """Return list of all matched workflows which are not completed
```

Comment:
Not true, it also writes in `extra_data`. That's surprising for a function called `get_...`.